Add patch to use 'gcc' instead of 'ld' for linking

The patch has been sent upstream but it has not been merged yet

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>

From 176ce06b40897a357fe76a558af517197325f188 Mon Sep 17 00:00:00 2001
From: Markos Chandras <markos.chandras@imgtec.com>
Date: Wed, 28 Aug 2013 10:56:32 +0100
Subject: [PATCH] rules/libobject.make: Use gcc for linking

Invoking 'ld' directly may miss some necessary linker flags which
are usually passed by gcc. Therefore, use gcc for linking.

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>
---
 rules/libobject.make | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rules/libobject.make b/rules/libobject.make
index bc670da..2cba658 100644
--- a/rules/libobject.make
+++ b/rules/libobject.make
@@ -3,7 +3,7 @@
 	if test -d $<.tmp; then rmdir $<.tmp; fi
 	mkdir $<.tmp
 	(cd $<.tmp && $(AR) x ../../$<)
-	$(LD) -o $@ -r $<.tmp/*.o
+	$(CC) -nostdlib -o $@ -r $<.tmp/*.o
 	rm -f $<.tmp/*.o && rmdir $<.tmp
 
 .PHONY: $(LTLIBRARIES:%.la=.libs/%.a)
-- 
1.8.3.2

